src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const checkSex = () => {
+    if (sex === 'm') {
+      return 'wife';
+    }
+
+    return 'husband';
+  };
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${checkSex()}`}
+        </p>
+      ) : (<p className="Person__partner">I am not married</p>)
+    }
```

Comment:
no need to repeat this code
```suggestion
        <p className="Person__partner">
           {isMarried ? (
             {`${partnerName} is my ${checkSex()}`}
           ) : (
             'I am not married'
           )
        </p>
      }
    }
```